src/kv/RocksDBStore.cc

```diff
@@ -449,8 +475,8 @@ void RocksDBStore::split_stats(const std::string &s, char delim, std::vector<std
 void RocksDBStore::get_statistics(Formatter *f)
 {
   if (!g_conf->rocksdb_perf)  {
-    dout(20) << __func__ << "RocksDB perf is disabled, can't probe for stats"
-	     << dendl;
+    dout(20) << __func__ << " RocksDB perf is disabled, can't probe for stats"
```

Comment:
space left

Reply:
thank you @amitkumar50 . It seems that we should left a space here. right?